components/application-operator/charts/application/templates/validator.yaml

```diff
@@ -37,6 +37,8 @@ spec:
             - "--eventServicePathPrefixV1=/{{ .Release.Name }}/v1/events"
             - "--eventServicePathPrefixV2=/{{ .Release.Name }}/v2/events"
             - "--eventServiceHost={{ .Release.Name }}-event-service:{{ .Values.eventService.service.externalapi.port }}"
+            - "--eventMeshPathPrefix=/{{ .Release.Name }}/events"
+            - "--eventMeshHost={{ .Release.Name }}.{{ .Release.Namespace }}"
```

Comment:
How is the traffic forwarded to Application Connectivity Validator? I do not see any match for `/{{ .Release.Name }}/events` path in https://github.com/kyma-project/kyma/pull/6422/files#diff-4b481e57ffc75a94ee22ebc3f342cd94R16-R22

Reply:
The traffic arrives to the validator as it always did. The only difference is that requests to `/myapp/events` are now handled by another reverse proxy, and sent to the Knative event mesh at `myapp.kyma-integration`.

Requests to `/v1` and `/v2` are still sent to the existing event-service, no change there.